README.md

```diff
@@ -12,6 +12,9 @@ chrome extension to queue youtube songs
 * An online store where you can share your playlists and find playlists of other users. 
 *Featured Videos. To display your own video in featured videos, email to deoraabhishek1995@gmail.com
 
+### Screenshots
```

Comment:
Why changes in Screenshots and contributors? Can you describe the changes that you made because I think the issue regarding these was already solved.

Reply:
Sir,It is the exact same changes.Yes,the issues were resolved but I was trying to align it in a better way but wasn't able to so left it as it is.

Reply:
Okay but the files is changed. Since you only made the previous changes also, I will merge it. But please add  only those files to the staging area which you wanna commit in case you change multiple files.